src/CallbackStream.php

```diff
@@ -160,7 +156,7 @@ public function getContents(): string
     /**
      * {@inheritdoc}
      */
-    public function getMetadata($key = null)
+    public function getMetadata(?string $key = null)
```

Comment:
Why isn't `mixed` return type used here (and other function with returned mixed) ?

Reply:
I see that it's because `mixed` type is not added for the interface method `StreamInterface::getMetaData()`. Then my question would be why wasn't it added there?

Reply:
To answer myself it's cause `php-fig/http-message` 2.0 is still PHP 7.2+ :man_facepalming:. https://github.com/php-fig/http-message/blob/2.0/composer.json#L14